jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/threadgroups/DslSetupThreadGroup.java

```diff
@@ -30,7 +30,15 @@
   public DslSetupThreadGroup(List<ThreadGroupChild> children) {
```

Comment:
maybe add name here (since we are adding it in other constructor), so is easily used here, and we keep constructor s consistent with other scenarios (if we enable setting name in one constructor, then we allow it in any of them). 

Also remember adding proper builder method in JmeterDsl as well, and pelase make similar changes to teardownThreadGroup to keep them consistent.

No need to update user guide.

Thank you very much.

Reply:
Oh sure. I'll add everything. :)